guides/client/js-interop.md

```diff
@@ -313,3 +319,42 @@ Hooks.Chart = {
 ```
 
 *Note*: In case a LiveView pushes events and renders content, `handleEvent` callbacks are invoked after the page is updated. Therefore, if the LiveView redirects at the same time it pushes events, callbacks won't be invoked on the old page's elements. Callbacks would be invoked on the redirected page's newly mounted hook elements.
+
+
+### Connecting multiple liveSockets
+
+LiveView allows connecting more than one `liveSocket`, each targeting different HTML nodes. This is useful to
+isolate the development cycle of a subset of the user interface. This means a different Phoenix application hosted
+in a different domain, can fully support an embedded LiveView. Think of it as Nested LiveViews, but instead of
+process-level isolation, it is a service-level isolation.
+
+Annotate your root views with a unique HTML attribute or class:
+
+```elixir
+# Main application serving a regular LiveView
+use GreatProductWeb.LiveView, container: {:div, "data-app": "root"}
+
+# Cats application, which will serve the cats component
+use CatsWeb.LiveView, container: {:div, "data-app": "cats"}
+```
+
+And initialise the liveSockets:
+
+```javascript
+# Fetch the disconnected render
+let disconnectedCatsHTML = await fetch("https://cats.io/live", { credentials: 'include' })
```

Comment:
would that be a controller doing a `live_render`, or do you actually have a `live "/live"` route in your router?